app/src/main/java/com/bnyro/recorder/ui/components/PlayerController.kt

```diff
@@ -0,0 +1,185 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.google.android.exoplayer2.ExoPlayer
+import com.google.android.exoplayer2.MediaItem
+import com.google.android.exoplayer2.Player
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
+import kotlinx.coroutines.launch
+
+@Composable
+fun PlayerController(exoPlayer: ExoPlayer) {
+    with(exoPlayer) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            val positionAndDuration by positionAndDurationState()
+            Row(Modifier.weight(1f), verticalAlignment = Alignment.CenterVertically) {
+                Text(DateUtils.formatElapsedTime(positionAndDuration.first / 1000))
+                var tempSliderPosition by remember { mutableStateOf<Float?>(null) }
+                Slider(
+                    modifier = Modifier.weight(1f),
+                    value = tempSliderPosition ?: positionAndDuration.first.toFloat(),
+                    onValueChange = { tempSliderPosition = it },
+                    valueRange = 0f.rangeTo(
+                        positionAndDuration.second?.toFloat() ?: Float.MAX_VALUE
+                    ),
+                    onValueChangeFinished = {
+                        tempSliderPosition?.let {
+                            exoPlayer.seekTo(it.toLong())
+                        }
+                        tempSliderPosition = null
+                    }
+                )
+                Text(
+                    positionAndDuration.second?.let { DateUtils.formatElapsedTime(it / 1000) }
+                        ?: ""
+                )
+            }
+            ElevatedCard(
+                colors = CardDefaults.elevatedCardColors(
+                    containerColor = MaterialTheme.colorScheme.tertiaryContainer,
+                    contentColor = MaterialTheme.colorScheme.onTertiaryContainer
+                ),
+                shape = CircleShape
+            ) {
+                val playState by isPlayingState()
+                IconButton(
+                    onClick = {
+                        playPause()
+                    }
+                ) {
+                    when (playState) {
+                        PlayerState.Buffer -> {
+                            CircularProgressIndicator(modifier = Modifier.size(24.dp))
+                        }
+
+                        PlayerState.Play -> {
+                            Icon(
+                                Icons.Default.Pause,
+                                contentDescription = "Pause"
+                            )
+                        }
+
+                        PlayerState.Pause -> {
+                            Icon(
+                                Icons.Default.PlayArrow,
+                                contentDescription = "Play"
```

Comment:
Please use a translatable string here, we should already have one for play I think.